rootfs/api/middleware.py

```diff
@@ -46,5 +46,7 @@ def process_response(self, request, response):
         a response header.
         """
         # clients shouldn't care about the patch release
-        response['DEIS_API_VERSION'] = __version__.rsplit('.', 1)[0]
+        version = __version__.rsplit('.', 1)[0]
+        response['DEIS_API_VERSION'] = version
+        response['DEIS_PLATFORM_VERSION'] = version
```

Comment:
Sorry, I forgot that the api version is short-form. I'd say that we should make PLATFORM_VERSION use `__version__` without the rsplit so we know the patch release running. For the API we don't really care about patch releases but it would be handy for `deis version --all` in the future.